scripts/validate-logging.js

```
@@ -0,0 +1,156 @@
+      VIOLATIONS.EXTERNAL_LOGGERS.push({
+        file: filePath,
+        pattern: pattern.toString(),
+      });
+    }
+  });
+}
+
+function printViolations() {
+  let hasViolations = false;
+
+  console.log('\n🔍 LOGGING VALIDATION REPORT\n');
```

ESLint

[error] Unexpected console statement.

(no-console)